packages/docusaurus-plugin-content-pages/src/index.ts

```diff
@@ -96,7 +97,7 @@ export default function pluginContentPages(
 
       function toMetadata(relativeSource: string): Metadata {
         const source = path.join(pagesDir, relativeSource);
-        const aliasedSourcePath = aliasedSitePath(source, siteDir);
+        const aliasedSourcePath = posixPath(aliasedSitePath(source, siteDir));
```

Comment:
I guess it's why CI build fails, because it's the only non-test code change related to pages?